src/main/java/Stream.java

```diff
@@ -2,7 +2,15 @@
 
 public class Stream {
     public static void main(String[] args){
-    List<String>list = List.of("aichi","kyoto","tokyo","osaka");
-    list.stream().map(String::toUpperCase).forEach(System.out::println);
+    List<String>lists = List.of("aichi","kyoto","tokyo","osaka");
```

Comment:
型と変数の間にスペース入れましょう！
```suggestion
    List<String> lists = List.of("aichi","kyoto","tokyo","osaka");
```

`prefectures`といったように中身がわかる変数名だと良いですね。

Reply:
型と変数の間にスペースを入れて訂正したものをもう一度PRさせていただきます。
変数につきましても訂正いたします。以前にもakamatsuさんから変数の指摘があったばかりですので、より注意を払いながら変数名を付けていこうと思います。